src/state/slices/opportunitiesSlice/resolvers/uniV2/utils.test.ts

```diff
@@ -93,13 +72,13 @@ describe('resolvers/univ2/utils', () => {
       token0Reserves: mockToken0Reserves,
     }
 
-    const totalDailyVolume = new BigNumber(amount0In).plus(
-      new BigNumber(amount0Out).div(1 - TRADING_FEE_RATE).decimalPlaces(0),
+    const totalDailyVolume = new BigNumber(mockAmount0In).plus(
+      new BigNumber(mockAmount0Out).div(1 - TRADING_FEE_RATE).decimalPlaces(0),
     )
 
     const totalAnnualVolume = totalDailyVolume.times(365.25)
     const totalAnnualRewards = totalAnnualVolume.times(TRADING_FEE_RATE).decimalPlaces(0)
-    const token0ReservesBaseUnitValue = new BigNumber(tokenAmount).times(`1e+${token0Decimals}`)
+    const token0ReservesBaseUnitValue = new BigNumber(tokenAmount)
```

Comment:
Looks weird but is correct - tests were previously off, confirmed at runtime we're still 🛍️